### PR TITLE
Added ground

### PR DESCRIPTION
**Pull request by Chris33871**
Added a more varied ground. Removed unused code and imports.
Is meant to be used in full screen. For it to work in half screen you would have to make the tanks tiny. It's dumb.

Please test thoroughly.

Don't bother saying you don't like the shape/would do it differently, I don't care. If you want a different ground, make it.

### CODE REVIEW

**Comment by 3NJDGZ** [comment]
how am i supposed to test this?

**Comment by 3NJDGZ** [comment]
when there's no logic to be tested? Its just visual ?

**Comment by Chris33871** [comment]
Just test that it works on your machine basically. If things are displayed properly and it doesn't break other pieces then we're good.

File: src/core/ground.js
Comment (3NJDGZ) [inline src/core/ground.js:1]: graphics looks good enough to me. It is responsive, but it distorts when you make the window's width smaller.
image
image
You see it gets 'distorted'? I cant lie, dont even bother lol

**Review by 3NJDGZ — APPROVED** [review]
ye looks good everything seems to be in order good job

File: src/core/ground.js
Comment (Chris33871) [reply src/core/ground.js:1]: Yeah that's exactly what I already said sherlock 🖕

Comment (3NJDGZ) [reply src/core/ground.js:1]: why tanks fly!!!

**Pull request merged**